File: con_with_most_water/Brute/water.cpp
class Solution {
public:
    int maxArea(vector<int>& height) {

        int n = height.size();

        int maxArea = 0;
        
        for(int i = 0; i<n; i++){
            for(int j= i+1; j<n; j++){
                int w = j-i;
                int current_height = min(height[i], height[j]);
                int area = w*current_height;

                maxArea = max(maxArea, area);
            }

        }
        return maxArea;

    }
};

File: con_with_most_water/Brute/water.py
class Solution:
    def maxArea(self, height: List[int]) -> int:
        n = len(height)
        maxArea = 0
        for i in range(n):
            for j in range(i+1,n):
                w = j-i
                curr_height = min(height[i],height[j])
                area = w*curr_height
                maxArea = max(area, maxArea)

        
        return maxArea
